### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by necessarily-equal**
You probably want to review the commits separately.

One is a pure formatting change, the other sorts and rename the weapons in the bot list so that they are named and sorted like usual.

See also https://github.com/UnvanquishedAssets/res-weapons_src.dpkdir/pull/7

### CODE REVIEW

File: ui/server_bots.rml [hunk 1]
```diff
@@ -72,37 +72,37 @@
...
+					<h3> Pain Saw </h3>
+				</row>
 				<row>
 					<input cvar="g_bot_shotgun" type="checkbox" />
 					<h3> Shotgun </h3>
 				</row>
 				<row>
 					<input cvar="g_bot_lasgun" type="checkbox" />
-					<h3> Lasgun </h3>
+					<h3> Las Gun </h3>
```
Comment (slipher) [inline ui/server_bots.rml:85]: Painsaw, Lasgun and Flamethrower are all one word.

**Comment by necessarily-equal** [comment]
I've copied it from what's printed in the armory. Should I change it 
there too?

**Comment by slipher** [comment]
Yes

**Comment by necessarily-equal** [comment]
ok, will work on this further

**Comment by necessarily-equal** [comment]
done

**Pull request merged**